modules/fastp/main.nf

```diff
@@ -34,7 +34,7 @@ process FASTP {
         [ ! -f  ${prefix}.fastq.gz ] && ln -s $reads ${prefix}.fastq.gz
```

Comment:
```suggestion
        [ ! -f  ${prefix}.fastq.gz ] && ln -sf $reads ${prefix}.fastq.gz
```

Reply:
Sorry to hijack your PR, but while you're changing stuff, this could be a nice addition. Currently, fastp jobs fail on retry because of the linking.

Reply:
what does the -sf do?

Reply:
It overwrites the existing symlink. Without the `-f` , `ln` throws an error if it already exists, causing the job to fail

Reply:
the other `ln` commands for paired mode should also have the `-f` added

Reply:
cool, i think there are some more modules with a similar setup, should probably update it everywhere then

Reply:
That might be advisable yes. Can be done as we encounter them